src/HomePage/components/MovieCard/MovieCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Popup from "../Popup/Popup";
+import './MovieCard.css'
+import '../Popup/Popup.css'
+import {useTogglerState} from "./Hooks";
+
+interface MovieCardInterface {
+    title: string,
+    popularity: number
+}
+
+export function MovieCard(props: MovieCardInterface) {
+    const [isOpen, toggleIsOpen] = useTogglerState(false);
+    const [isEdited, toggleIsEdited] = useTogglerState(false);
+    const [isDeleted, toggleIsDeleted] = useTogglerState(false);
+
+    const togglePopup = () => {
+        toggleIsOpen();
+    }
+
+    const toggleEdit = () => {
+        toggleIsEdited();
+    }
+
+    const toggleDelete = () => {
+        toggleIsDeleted();
+    }
```

Comment:
You don't need to wrap toggleIsDelete function call into another function - just use it directly, the same for toggleIsEdited, toggleIsOpen